internal/wclayer/cim/process.go

```diff
@@ -7,95 +7,16 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
-	"syscall"
 	"time"
 
 	"github.com/Microsoft/go-winio"
-	"github.com/Microsoft/go-winio/vhd"
-	"github.com/Microsoft/hcsshim/computestorage"
-	"github.com/Microsoft/hcsshim/internal/memory"
-	"github.com/Microsoft/hcsshim/internal/security"
-	"github.com/Microsoft/hcsshim/internal/vhdx"
 	"github.com/Microsoft/hcsshim/internal/wclayer"
 	"golang.org/x/sys/windows"
 )
 
-const defaultVHDXBlockSizeInMB = 1
-
-// processUtilityVMLayer is similar to createContainerBaseLayerVHDs but along with the scratch creation it
-// also does some BCD modifications to allow the UVM to boot from the CIM. It expects that the UVM BCD file is
-// present at layerPath/`wclayer.BcdFilePath` and a UVM SYSTEM hive is present at
-// layerPath/UtilityVM/`wclayer.RegFilesPath`/SYSTEM. The scratch VHDs are created under the `layerPath`
-// directory.
+// processUtilityVMLayer will handle processing of UVM specific files when we start
+// supporting UVM based containers with CimFS in the future.
 func processUtilityVMLayer(ctx context.Context, layerPath string) error {
```

Comment:
I only removed the function body because removing the entire function means we will have to make some more changes to the caller locations too and since this PR is already too big I didn't want to add more changes here. I can get rid of it in the next PR, is that okay?